linter.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
Can you create a `scripts` folder to put this linter in? That will help keep the code files organized

Reply:
done